test/checks/visibility/hidden-content.js

```diff
@@ -1,46 +1,81 @@
+/* global xit */
 describe('hidden content', function () {
 	'use strict';
 
-  var fixture = document.getElementById('fixture');
-
+	var fixture = document.getElementById('fixture');
+	var shadowSupport = document.body && typeof document.body.attachShadow === 'function';
 	var checkContext = {
 		_data: null,
 		data: function (d) {
 			this._data = d;
 		}
 	};
 
+	function checkSetup (html, options, target) {
```

Comment:
Not sure. I like having some test utilities to work with, but I don't much like just dropping a bunch of extra functions on the global object. I suppose we could create a `/test/utils` dir, and have a global `testUtils` object that has everything on it. Something like that?